src/Frame.js

```diff
@@ -0,0 +1,52 @@
+function Frame() {
+  this.turn = 0
+  this.pins = 10
+  this.currentFrame = []
+  this.metaFrame = []
+  this.strike = false
+  this.spare = false
+}
+
+Frame.prototype.play = function(hitPins) {
```

Comment:
This function has too many statements. (16)<br>This function's cyclomatic complexity is too high. (6)